core/src/com/group66/game/screens/GameScreen.java

```diff
@@ -53,8 +53,11 @@ public GameScreen(BustaMove game, Boolean randomLevel) {
 		setup_keys();
 		AssetLoader.load();
 
-		LevelLoader.loadLevel(ballManager);
-		//LevelLoader.generateLevel(ballManager);
+		if (!randomLevel) {
+		    LevelLoader.loadLevel(ballManager);
+		} else {
+		    LevelLoader.generateLevel(ballManager);
+		}
```

Comment:
In the main menu, you can choose between level1 and random. This handles the option.